x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/search_experience.scss

```
@@ -25,4 +25,20 @@
     background-color: $euiPageBackgroundColor;
     padding: $euiSizeL;
   }
+  
+  .documentsSearchExperience__facet {
+    .euiCheckboxGroup__item {
+      // Adding text-overflow to euiCheckbox__label below
+      // causes approximately 3px of bottom margin. this corrects
+      // that.
+      margin-bottom: -3px;
+    }
```

[optional, not a change request] Nice catch! It looks like you could also accomplish a similar compression by squashing line height:

```suggestion
  .documentsSearchExperience__facet {
    line-height: 0;
```

Totally your call though, I think line-height might be a little more flexible/responsive than flat px

---

https://github.com/elastic/kibana/pull/88491/commits/ba3c3d2338856b9e5384fbf47ea2d23fa728d938